api.php

```diff
@@ -411,13 +425,12 @@
 	// if that id is not a number or is 0
 	if (!$id) {
 		// set a message to explain what broke
-		$response->getBody()->write(json_encode([
+		return badRequestResponse([
 			'field' => ['id'],
 			'message' => 'invalid id provided',
-		]));
-		// return the error and a invalid request status
-		return $response->withStatus(400);
+		], $response);
 	}
+
```

Comment:
if i try to find a review by id that dose not exist there is not error message for that.

Reply:
also out of scope, add for review routes